src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -455,6 +491,55 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
         return false;
     }
 
+    getMutationPlotControls() {
+        const length = this.props.store.getMutationStore(
+            this.props.chartMeta.uniqueKey
+        ).samplesViaSelectedCodons.length;
+
+        return (
+            <div
+                style={{
+                    position: 'absolute',
+                    bottom: 5,
+                    display: 'flex',
+                }}
+            >
+                <button
+                    className={classnames(
+                        styles.controls,
+                        styles.controlsAtTop
+                    )}
+                    disabled={length == 0}
+                    onClick={_ =>
+                        length > 0
+                            ? this.props.store.updateStudyViewFilter(
+                                  this.props.chartMeta.uniqueKey
+                              )
+                            : () => {}
+                    }
+                >
+                    {`Select ${length} ${length == 1 ? 'sample' : 'samples'}`}
```

Comment:
use this function instead https://github.com/cBioPortal/cbioportal-frontend/blob/5e9f19863ebc953c67dbe561408149a5e6488c16/packages/cbioportal-frontend-commons/src/lib/StringUtils.ts#L59

```suggestion
                    {`Select ${length} ${pluralize('sample', length)}`}
```